src/main/webapp/app/overview/course-lectures/course-lectures.component.ts

```diff
@@ -23,6 +23,14 @@ const DEFAULT_COLLAPSE_STATE: CollapseState = {
     noDate: true,
 };
 
+const DEFAULT_ALWAYS_SHOW: SidebarItemShowAlways = {
+    future: false,
+    current: false,
+    dueSoon: false,
+    past: false,
+    noDate: false,
+};
```

Comment:
This appears to be a duplicate with the definition in `src/main/webapp/app/overview/course-exercises/course-exercises.component.ts` does it make sense to have it shared with the exercise definition?